src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.Composite.sfxproj

```diff
@@ -33,6 +33,25 @@
     <PartialCompositeAssemblyListPath>PartialCompositeAssemblyListPath.txt</PartialCompositeAssemblyListPath>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Internal.Runtime.AspNetCore.Transport"
+        Version="$(MicrosoftInternalRuntimeAspNetCoreTransportVersion)"
+        ExcludeAssets="All"
+        GeneratePathProperty="true" />
+  </ItemGroup>
+
+  <!-- Explicitly include symbols for libraries we redistribute from runtime -->
+  <ItemGroup>
+    <_TransportPdbs Include="$(PkgMicrosoft_Internal_Runtime_AspNetCore_Transport)\**\*.pdb"
+        Exclude="$(PkgMicrosoft_Internal_Runtime_AspNetCore_Transport)\**\*.ni.pdb" />
```

Comment:
Condition so it doesn't enumerate the whole filesystem during the first restore (before the path property is available).

```suggestion
    <_TransportPdbs Include="$(PkgMicrosoft_Internal_Runtime_AspNetCore_Transport)\**\*.pdb"
        Exclude="$(PkgMicrosoft_Internal_Runtime_AspNetCore_Transport)\**\*.ni.pdb"
        Condition="'$(PkgMicrosoft_Internal_runtime_AspNetCore_Transport)' != '' />
```